### PR TITLE
Cockpit 299

### PR DESCRIPTION
**Pull request by tomasmatus**

### CODE REVIEW

File: _posts/2023-08-23-cockpit-299.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+![Screenshot from 2023-08-23 11-17-45]({{ site.baseurl }}/images/299-kdump-show-location-of-kdump-to-verify-the-successful-configuration-test.png)
+
+## Storage: Support for no-overprovisioning with Stratis
+
+A Stratis pool can be put into a "no-overprovisioning" mode where the bad effects of running out
+of space can be avoided via careful management of filesystem sizes. Cockpit now supports this mode.
+
+## Storage: Cockpit can now add caches to encrypted Stratis pools
+
+Encrypted caches are a new feature in Stratis 3.5, and now you can use it also from Cockpit.
```
Comment (jelly) [inline _posts/2023-08-23-cockpit-299.md:30]: These two don't have a screenshot, do we want to add one?

File: _posts/2023-08-23-cockpit-299.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Cockpit 299 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/299)
+* [Cockpit Fedora 40](https://bodhi.fedoraproject.org/updates/FEDORA-2023-4d17532ade)
+* [Cockpit Fedora 39](https://bodhi.fedoraproject.org/updates/FEDORA-2023-b19a60b701)
+* [Cockpit Fedora 38](https://bodhi.fedoraproject.org/updates/FEDORA-2023-bbb476ce91)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-1ab32d4881)
```
Comment (jelly) [inline _posts/2023-08-23-cockpit-299.md:44]: We don't have to show F39/F40 these are not released yet.

Comment (tomasmatus) [reply _posts/2023-08-23-cockpit-299.md:42]: ok, removed

File: _posts/2023-08-23-cockpit-299.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 299
+author: tmatus
+date: '2023-08-23'
+tags: cockpit
+slug: cockpit-299
+category: release
+summary: ''
```
Comment (jelly) [inline _posts/2023-08-23-cockpit-299.md:8]: This needs to be filled in.

Comment (tomasmatus) [reply _posts/2023-08-23-cockpit-299.md]: Ah, probably got lost when regenerating :)

**Review by martinpitt — CHANGES_REQUESTED** [review]
Bwah, I created this review two days ago, and forgot to send it.

File: _posts/2023-08-23-cockpit-299.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 299:
+
+
+## Storage: Cockpit can now add caches to encrypted Stratis pools
+
+Encrypted caches are a new feature in Stratis 3.5, and now you can use it also from Cockpit.
```
[the remark this answers is not in the document]
Comment (martinpitt) [reply _posts/2023-08-23-cockpit-299.md:30]: ... at this point it should be edited inline here, not in the PR any more.

[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2023-08-23-cockpit-299.md:30]: So, the user visible part of this change is that the "Cache" option in the tier dropdown is no longer disabled:

image

Stratis release notes that talk about this: https://stratis-storage.github.io/stratis-release-notes-3-5-0/

**Review by martinpitt — APPROVED** [review]
Thank you!

**Pull request merged**